.travis.yml

```diff
@@ -1,5 +1,12 @@
 language: ruby
 rvm:
-  - 2.3.4
   - 2.4.1
   - 2.3.1
```

Comment:
We should remove 2.3.1 since we're all using 2.4.1. Other than that, looks good.